analyzer/tools/build-logger/Makefile.manual

```diff
@@ -115,4 +115,4 @@ ldlogger_64.so: $(LDLOGGER_LIB_SOURCES) $(LDLOGGER_LIB_HEADERS)
 	$(CC) -m64 $(CPPFLAGS) $(LIB_CFLAGS) $(LIB_LDFLAGS) $(LDLOGGER_LIB_SOURCES) $(LDLOGGER_SOURCES) -o $@
 
 test:
-	test/test_logger.sh $(BUILD_DIR)
+	test/test_logger.py $(BUILD_DIR)
```

Comment:
It would be better to follow the same structure every in this repository. For this reason I recommend you to check the structure of the statistics-collector tool first: https://github.com/Ericsson/codechecker/tree/master/analyzer/tools/statistics_collector

Next do the following things:
- Create a `Makefile` in the test directory of the logger tool where you will have almost the same thing as here: https://github.com/Ericsson/codechecker/blob/master/analyzer/tools/statistics_collector/tests/Makefile. You don't have to add pylint and pycodestyle for this now.
- Run the test with nose. For this add it to the build logger requirements: https://github.com/Ericsson/codechecker/blob/a6b3f785adaf50b16039814ea222ae4b6b2fc0d9/analyzer/tools/statistics_collector/requirements_py/dev/requirements.txt#L1
- Add new targets to create the venv for this to run tests similar to this:
https://github.com/Ericsson/codechecker/blob/a6b3f785adaf50b16039814ea222ae4b6b2fc0d9/analyzer/tools/statistics_collector/Makefile#L25-L32
- Also the directory structure of the test should be the following:
  ```sh
  |- tests
    |- unit
    |- functional  # I think for ld-logger it is a functional test.
    |- Makefile
  ```
Create a Makefile under the functional directory and in the tests/Makefile you just have to include the Makefile from the test folder similar to this:
https://github.com/Ericsson/codechecker/blob/a6b3f785adaf50b16039814ea222ae4b6b2fc0d9/analyzer/tools/statistics_collector/Makefile#L37